package.json

```diff
@@ -12,6 +11,11 @@
     "publish": "npm run clean && npm run build && lerna publish -m \"Publish npm packages\""
```

Comment:
Why do these scripts still use `npm run clean`, `npm run build`? Should they be changed to `yarn run clean` or `yarn run build`?